.github/workflows/inspecjs-tests.yml

```diff
@@ -0,0 +1,53 @@
+name: Run InspecJS Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    services:
+      postgres:
+        image: postgres:latest
+        env:
+          POSTGRES_PASSWORD: postgres
+        ports:
+          - 5432:5432
+
+    steps:
+     - uses: actions/checkout@v2
+
+     - name: Cache node modules
+       uses: actions/cache@v2
+       env:
+         cache-name: cache-node-modules
+       with:
+         # npm cache files are stored in `~/.npm` on Linux/macOS
+         path: ~/.npm
+         key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
```

Comment:
We're using yarn which generates a `yarn.lock` not a `package-lock.json`, this is wrong in all of the other workflows as well unfortunately.